kolibri/core/content/api.py

```diff
@@ -373,9 +387,17 @@ def filter_options(self, request, **kwargs):
         return Response(data)
 
 
+def _create_channel_thumbnail_url(item):
+    return reverse("kolibri:core:channel-thumbnail", args=[item["id"]])
```

Comment:
It looks like we are still loading the `thumbnail` data from the database - perhaps we could also do a falsey check here?

```
return reverse("kolibri:core:channel-thumbnail", args=[item["id"]]) if item["thumbnail"] else ""
```

That would mean that we don't set a URL in the case that the thumbnail is empty, and Kolibri can display the default channel icon instead of the thumbnail (whereas if the thumbnail URL 404s, we will see a broken image display in the browser).